src/CattyTests/Formula/FormulaTest.swift

```diff
@@ -180,4 +180,58 @@ final class FormulaTest: XCTestCase {
         formula = Formula(formulaElement: parseTree)!
         XCTAssertFalse(formula.isSingularNumber())
     }
+
+    func testGetDisplayString() {
+
+        var formulaString = "1.0"
+        var formula = Formula(string: formulaString)!
+        var formattedString = String(format: "'%@'", formulaString)
+        var displayString = formula.getDisplayString()
+        XCTAssertEqual(displayString, formattedString)
+
+        formulaString = "10"
```

Comment:
Please add a test for the Integer formula type and one for tested Formulas (e.g., using the add operator and a function). All your tests are still testing a String data type only (you can remove them). Please see the other initializers of the Formula.